aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -294,14 +337,18 @@ module AptosFramework::Token {
     }
 
     public fun initialize_token_store(account: &signer) {
-        move_to(
-            account,
-            TokenStore {
-                tokens: Table::new(),
-                deposit_events: Event::new_event_handle<DepositEvent>(account),
-                withdraw_events: Event::new_event_handle<WithdrawEvent>(account),
-            },
-        );
+        if (!exists<TokenStore>(Signer::address_of(account))) {
```

Comment:
I don't think this is necessary, if someone calls initialize twice a failed transaction is fine.

Reply:
If we don't have this. The txn failed and error message is hard to read and user may not know why their txn failed. 
with this check, their txn will just succeeded without causing confusion on why their txn failed